src/environments/environment.stg.ts

```diff
@@ -23,5 +23,6 @@ export const environment = {
   SENTRY_ENVIRONMENT: 'staging',
   idleTime: 900,
   idleTimeout: 300,
+  orgRequestEmail: 'john.henderson@energyweb.org',
```

Comment:
add this line to the environment.prod.ts (for release branch), as this stg is for staging (for master branch).

Reply:
@dawidgil its already in prd